symplyphysics/core/vectors/arithmetics.py

```diff
@@ -196,7 +203,8 @@ def reject_cartesian_vector(
     )
 
 
-def diff_cartesian_vector_components(
+# Only works in non-rotating coordinate systems
+def diff_cartesian_vector(
```

Comment:
Do we really need multiple diff variables? I think one should be enough for most cases.

Reply:
In my experience it's quite often that we need several variables.

Reply:
I think it should give the same result, when taking derivative of one variable multiple times. So I believe we can make code easier this way.